src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -655,6 +655,8 @@ Engine::LoadExeNetworkImpl(const InferenceEngine::CNNNetwork &network, const std
     }
 
     auto config = orig_config;
+    config.insert(engConfig._config.begin(), engConfig._config.end());
```

Comment:
Do we really need configs merge right now? As I can see `modelCacheProp` checks both `config` and `engConfig`. 
I agree that merge itself sounds like a better solution, but would like to avoid possible side affects right before CF.
@EgorDuplensky what is your opinion here?

Reply:
We definitely shouldn't do this.
Basically we override user's config for ov::CompiledModel with engine's (ov::Core) config.
And actually it should be the opposite, CompiledModel config should have more priority in general (it's ensured on line 681-683, where we read CompiledModel config on top of Engine config).
Also It's not good that this does not result in any failed test.
And I can assume this is not even documented anywhere.

Reply:
Also we should not do the opposite insertion, like
`engConfig._config.insert(config.begin(), config.end());`
because this way we would override Engine config.

Reply:
Config merging removed.